core/arch/aarch64/instr_create.h

```diff
@@ -45,61 +45,50 @@
  * Used in an additional immediate source operand to a vector operation, denotes
  * 8 bit vector element width. See \ref sec_IR_AArch64.
  */
-#define ISZ_BYTE 0
+#define BYTE_SZ 0
```

Comment:
Since these are in the global namespace, it seems best to have a more unique name.  How about VECTOR_SIZE_BYTE?  Yes it's longer but it's clear and less likely to collide.  Similarly for the others.